runtime/lf_skiplist.c

```diff
@@ -198,7 +198,7 @@ static int skiplist_find(struct lf_skiplist *sk, uintnat key,
           thread can ever add the node to the garbage list. This is what the
           compare-and-swap below ensures by swapping garbage_next to a value
           of 1. We don't need to worry about anyone accidentally following this
-          bogus pointer, it is only deferenced in the cleanup function and this
+          bogus pointer, it is only dereferenced in the cleanup function and this
           is called when no thread can be concurrently modifying the skiplist.
```

Comment:
Oh dear, sorry for two round-trips on a one-word change, but now our `tools/check-typo` script is unhappy because the line is now more than 80 characters long! (more info can be found in [HACKING.adoc](https://github.com/ocaml/ocaml/blob/trunk/HACKING.adoc#check-typo))
```suggestion
          bogus pointer, it is only dereferenced in the cleanup function and
          this is called when no thread can be concurrently modifying the
          skiplist.
```